mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,53 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import styles from "src/component/input/Input.module.scss";
+
+/**
+ * The `InputProps` interface represents a set of properties that can be passed to the `Input` component.
```

Comment:
Input's properties will be enough. No need very long commentswithout useful info

Reply:
Don't forget to use {@ link)